docs/cli/graphql-transformer/directives.md

```diff
@@ -476,60 +476,66 @@ type Post
 }
 ```
 
-Owner authorization specifies that a user can access an object. To
-do so, each object has an *ownerField* (by default "owner") that stores ownership information
-and is verified in various ways during resolver execution.
+Owner authorization specifies that a user can access an object. To do so, each object will get an `ownerField` field (by default "owner", will be added to the object if not specified) that stores ownership information and is verified in various ways during resolver execution.
 
-You can use the *operations* argument to specify which operations are augmented as follows:
+You can use the `operations` argument to specify which operations are enabled as follows:
 
 - **read**: If the record's owner is not the same as the logged in user (via `$ctx.identity.username`), throw `$util.unauthorized()` in any resolver that returns an object of this type.
-- **create**: Inject the logged in user's `$ctx.identity.username` as the *ownerField* automatically.
-- **update**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
-- **delete**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
+- **create**: Inject the logged in user's `$ctx.identity.username` as the `ownerField` automatically.
+- **update**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
+- **delete**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
 
-**Note**: When specifying operations as a part of the @auth rule, the operations not included in the list are not protected by default. For example, let's say you have the following schema:
+**Note**: When specifying operations as a part of the `@auth` rule, the operations not included in the list are not protected by default.
+
+Let's take a look at a few examples:
 
 ```graphql
 type Todo @model
-  @auth(rules: [{ allow: owner, operations: [read] }]) {
+  @auth(rules: [{ allow: owner }]) {
   id: ID!
   updatedAt: AWSDateTime!
   content: String!
 }
 ```
 
-In this schema, only the owner of the object has the authorization to perform read (getTodo and listTodos) operations on the owner created object. But this does not prevent any other owner (any user other than the creator or owner of the object) to update/delete some other owner's object.
+In this schema, only the owner of the object has the authorization to perform read (`getTodo` and `listTodos`) and update operations (`updateTodo` and `deleteTodo`) on the owner created object. This prevents any other owner (any user other than the creator of the object) to update/delete some other owner's object.
+
 Here's a truth table for the above-mentioned schema. In the table below `other` refers to any user other than the creator or owner of the object.
```

Comment:
Idle Thought: "or" is kind of a weird word here. I get it's a truth table, but I don't think that fact is pulling much weight in this sentence in terms of helping me understand. Maybe...

> Here's a table outlining which users are permitted to execute which operations. `Other` means all users, `owner` refers to the user who created the object.

Say, why don't we just use "All Users" instead of "other?"

Reply:
Agreed, I've made some updates here. I think "All users" is misleading because owners would be included here, instead I think we need to explain what "other" means or have another better word. I used this: 

 **owner** refers to the user who created the object,  **other** refers to all other users.